Detectors/GlobalTracking/src/MatchITSTPCQC.cxx

```diff
@@ -101,6 +119,13 @@ bool MatchITSTPCQC::init()
   mPhiTPCPhysPrim->Sumw2();
   mPhiPhysPrim->Sumw2();
 
+  mPtTPC->SetOption("logy");
```

Comment:
This will not give you the log scale as we discussed, right?

Reply:
No, the log binning is set between L92-L107 in the updated version